_i18n/de/resources/moneropedia/daemon.md

```diff
@@ -1,16 +1,16 @@
 ---
-terms: ["daemon"]
-summary: "Background process which runs and controls a Monero node"
+terms: ["daemon", "Hintergrunddienst"]
+summary: "Hintergrundprozess, der einen Monero-Node betreibt und steuert"
 ---
 
-{% include disclaimer.html translated="no" translationOutdated="no" %}
-'Daemon' is the general term for a piece of software running in the background. In Monero, the Daemon is started through the 'monerod' program. If you run the Daemon locally, you are running a local @node. If the Daemon is running on another device it's a @remote-node. A @wallet, like the CLI or the GUI, needs to connect to a Daemon (local or remote) to relay @transactions to the network.
+{% include disclaimer.html translated="yes" translationOutdated="no" %}
+"Hintergrunddienst" ist der allgemeine Ausdruck für einen Teil einer Software, welcher im Hintergrund läuft. Bei Monero wird der Hintergrunddienst durch das "Monerod"-Programm gestartet. Wenn du den Hintergrunddienst lokal laufen lässt, betreibst du einen lokalen @Node. Wenn der Hintergrunddienst auf einem anderen Gerät läuft, handelt es sich um einen @Drittanbieter-Node. Ein @Wallet (wie CLI oder GUI) muss zu einem (lokalen oder Drittanbieter-) Node verbinden, um dem Netzwerk Transaktionen zu übermitteln.
```

Comment:
I saw this translation being used on Weblate and I thought it was technically correct, it made sense to me. Pinging @erciccione for some input; otherwise I'd be fine with translating it back to "Remote-Node".

Reply:
Whatever works for you is fine. Remote is a generic word, so probably makes sense to translate it.

> Does it make sense to specify that the wallet connects itself to the node

Not necessary in my opinion.

Reply:
> I saw this translation (Drittanbieter-Node) being used on Weblate and I thought it was technically correct, it made sense to me.

* "Remote-Node" = a node run "far away" (not on the same computer)
* "Drittanbieter-Node" = "third party node" = a node run and owned by someone else

Every "third party node" is a "remote node", but not every "remote node" is a "third party node", since many users run their own "remote nodes" (which is even recommended since it improves security).

Therefore the translation as "Drittanbieter-Node" ("third party node") seems incorrect.

I still haven't come up with a better translation though...

* "remote" = "fern"
* "remote control" = "Fernsteuerung"
* "remote maintenance" = "Fernwartung"
* "remote node" = ?

> Does it make sense to specify that the wallet connects itself to the node as you did with adding "sich"?

In German yes, in English no. :)

"a client connects to a server" would be "Ein Client verbindet sich zu einem Server".. doesn't work without "sich".

You could also translate it as "Ein Client baut eine Verbindung zu einem Server auf".. but that'd rather be "A client establishes a connection to a server" in English.

> And I used "Software" throughout the German translation since it's pretty much used,

I guess it depends on the context - "software" and "program" cannot be interchanged in many cases.

Wikipedia on "computer program": "A collection of computer programs, libraries, and related data are referred to as software."

> but I can remove "piece of"/"Teil einer" if it's better?

First what I had in mind is Wikipedia's definition for "daemon":

"a daemon is a computer program that runs as a background process".

(short and simple.. it's the whole program, not part of it.. also note that it uses "program" instead of "software")

But after reading this paragraph again I'd actually keep your translation ("Teil einer Software"), because it is more detailed and technically correct:

A program can spawn one (or even more) daemons that keep running in the background, while the same program can provide other functions that run in the foreground, depending on how the program is parameterized. So the daemon actually is only part of the program.

Reply:
Your explanation makes absolutely sense. Following your approach we could decide to translate it as "Fern-Node"/"Fernnode", but personally I would prefer "Remote-Node" in that case. I don*t need to translate a term come hell or high water just to have a German-only translation (which isn't really possible in this field anyway). Maybe "Remote-Node would be self-explanatory enough..?

> "a client connects to a server" would be "Ein Client verbindet sich zu einem Server".. doesn't work without "sich".

I believed that "sich" is rather used in common, everyday use of language in these cases. Personally, I like the version "baut eine Verbindung zu X auf". Would that be a fine compromise?

Reply:
> Maybe "Remote-Node" would be self-explanatory enough..?

+1 for leaving it as "Remote-Node" since I never heard the word "Fernnode" or "Fernknoten" being used.

> Personally, I like the version "baut eine Verbindung zu X auf". Would that be a fine compromise?

Yes! "Ein @Wallet (wie CLI oder GUI) baut eine Verbindung zu einem (lokalen oder Remote-) Node auf, um dem Netzwerk Transaktionen zu übermitteln."

Reply:
Well, then I'm gonna go to work, hopefully I'll find all occurrences to change! :)